# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/ipc/CanvasTranslator.h
# Commit: 57fc38773fe3
# Full Hash: 57fc38773fe37f21748092e3e58fdf5265e85c0d
# Author: Bob Owen <bobowencode@gmail.com>
# Date: 2023-08-30 04:26:43
# Regressor Bug: 1841891
# File Overlap Count: 3
# Description:
#   Bug 1841891: Only use a larger canvas ring buffer when in the foreground. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D186544
# ==============================================================================

diff -r 082442e3b925 -r 57fc38773fe3 gfx/layers/ipc/CanvasTranslator.h
--- a/gfx/layers/ipc/CanvasTranslator.h	Tue Aug 29 19:15:34 2023 +0000
+++ b/gfx/layers/ipc/CanvasTranslator.h	Tue Aug 29 19:17:45 2023 +0000
@@ -63,6 +63,11 @@
       CrossProcessSemaphoreHandle&& aWriterSem);
 
   /**
+   * New buffer to resume translation after it has been stopped by writer.
+   */
+  ipc::IPCResult RecvNewBuffer(ipc::SharedMemoryBasic::Handle&& aReadHandle);
+
+  /**
    * Used to tell the CanvasTranslator to start translating again after it has
    * stopped due to a timeout waiting for events.
    */